### PR TITLE
support background image and remote ice servers for avatar

### PR DESCRIPTION
**Pull request by yulin-li**

### CODE REVIEW

**Review by yinhew — APPROVED** [review]

File: src/sdk/AvatarConfig.ts [hunk 1]
```diff
@@ -43,19 +45,49 @@ export class AvatarConfig {
...
+    }
+
+    /**
+     * Sets the remote ICE servers.
+     * @remarks Normally, the ICE servers are gathered from the PeerConnection,
+     * set this property to override the ICE servers. E.g., the ICE servers are
+     * different in client and server side.
+     * @param {RTCIceServer[]} value - The remote ICE servers.
+     */
+    public set remoteICEServers(value: RTCIceServer[]) {
```
Comment (yinhew) [inline src/sdk/AvatarConfig.ts:87]: Can you rename remoteICEServers to remoteIceServers to have consistent naming with others?

**Review by rhurey — APPROVED** [review]

File: src/sdk/AvatarConfig.ts [hunk 2]
```diff
@@ -43,19 +45,49 @@ export class AvatarConfig {
...
+
+    /**
+     * Sets the background image.
+     * @param {URL} value - The background image.
+     */
+    public set backgroundImage(value: URL) {
+        this.privBackgroundImage = value;
+    }
+
+    public get remoteIceServers(): RTCIceServer[] {
```
Comment (BrianMouncer) [inline src/sdk/AvatarConfig.ts:76]: No docs comment on this one?

**Review by BrianMouncer — APPROVED** [review]

**Comment by BrianMouncer** [comment]
close and reopen to kick a new build.

**Comment by BrianMouncer** [comment]
close and reopen to kick a new build.

**Comment by BrianMouncer** [comment]
ok, I fixed the translation test, and skipped the ambiguous speech test for now.
Your PR now has a green build.
I did make one comment about missing method doc comments...

File: src/sdk/AvatarConfig.ts (hunk 2 above)
Comment (yulin-li) [reply src/sdk/AvatarConfig.ts:76]: thanks for noticing this. I updated it

**Pull request merged**